### PR TITLE
docs: sync russian version of readme up to 'Build from source'

### PR DESCRIPTION
**Pull request by id**

### CODE REVIEW

File: README-RU.md [hunk 1]
```diff
@@ -7,39 +7,84 @@
...
+Для получения дополнительной информации, пожалуйста, посетите [домашнюю страницу EMQX](https://www.emqx.io/).
 
-## Установка
+## Начало работы
 
-Брокер *EMQX* кросплатформенный, и поддерживает Linux, Unix, macOS и Windows. Он может работать на серверах с архитектурой x86_64 и устройствах на архитектуре ARM, таких как Raspberry Pi.
+#### EMQX Cloud
 
-Более подробная информация о запуске на Windows по ссылке: [Windows.md](./Windows.md)
+Самый простой способ запустить EMQX это развернуть его с помощью EMQX Cloud. Вы можете [попробовать EMQX Cloud бесплатно](https://www.emqx.com/en/signup?utm_source=github.com&utm_medium=referral&utm_campaign=emqx-readme-to-cloud&continue=https://cloud-intl.emqx.com/console/deployments/0?oper=new), данные кредитной карточки не требуются..
```
Comment (savonarola) [inline README-RU.md:26]:
```suggestion
Самый простой способ запустить EMQX это развернуть его с помощью EMQX Cloud. Вы можете [попробовать EMQX Cloud бесплатно](https://www.emqx.com/en/signup?utm_source=github.com&utm_medium=referral&utm_campaign=emqx-readme-to-cloud&continue=https://cloud-intl.emqx.com/console/deployments/0?oper=new), данные кредитной карточки не требуются.
```

File: README-RU.md [hunk 2]
```diff
@@ -7,39 +7,84 @@
...
+
+## Участие в разработке
+
+Пожалуйста, прочитайте [contributing.md](./CONTRIBUTING.md).
+
+Для более организованных предложений по улучшению вы можете отправить pull requests в [EIP](https://github.com/emqx/eip).
+
+## Присоединяйтесь к коммьюнити
+
+- Следуйте [@EMQTech on Twitter](https://twitter.com/EMQTech).
```
Comment (savonarola) [inline README-RU.md:66]: "Следите за"?

File: README-RU.md [hunk 3]
```diff
@@ -7,39 +7,84 @@
...
+- Подключайтесь к [обсуждениям](https://github.com/emqx/emqx/discussions) на Github если у вас есть какой-то вопрос.
```
Comment (savonarola) [inline README-RU.md:67]:
```suggestion
- Подключайтесь к [обсуждениям](https://github.com/emqx/emqx/discussions) на Github, если у вас есть какой-то вопрос.
```

File: README-RU.md [hunk 4]
```diff
@@ -7,39 +7,84 @@
...
+- Присоединяйтесь к нашему [официальному Discord](https://discord.gg/xYGf3fQnES) чтобы поговорить с командой разработки.
```
Comment (savonarola) [inline README-RU.md:68]:
```suggestion
- Присоединяйтесь к нашему [официальному Discord](https://discord.gg/xYGf3fQnES), чтобы поговорить с командой разработки.
```

File: README-RU.md [hunk 5]
```diff
@@ -7,39 +7,84 @@
...
+- Подписывайтесь на канал [EMQX YouTube](https://www.youtube.com/channel/UC5FjR77ErAxvZENEWzQaO5Q).
+
+## Дополнительные ресурсы
+
+- [MQTT client programming](https://www.emqx.com/en/blog/tag/mqtt-client-programming)
+
+  Коллекция блогов чтобы помочь разработчикам быстро начать работу с MQTT на PHP, Node.js, Python, Golang, и других языках программирования.
```
Comment (savonarola) [inline README-RU.md:75]:
```suggestion
  Коллекция блогов, чтобы помочь разработчикам быстро начать работу с MQTT на PHP, Node.js, Python, Golang, и других языках программирования.
```

File: README-RU.md [hunk 6]
```diff
@@ -7,39 +7,84 @@
...
+- [MQTT X](https://mqttx.app/)
 
-Сборки для различных операционных систем: [Загрузить EMQX](https://www.emqx.com/en/downloads).
+  Элегантный кроссплатформенный клиент MQTT 5.0, в виде десктопного приложения, приложения для командной строки и веб-приложения, чтобы помочь вам быстрее разрабатывать и отлаживать службы и приложения MQTT.
 
-- [Установка на одном сервере](https://www.emqx.io/docs/en/latest/getting-started/install.html)
-- [Установка на кластере](https://www.emqx.io/docs/en/latest/advanced/cluster.html)
+- [Internet of Vehicles](https://www.emqx.com/en/blog/category/internet-of-vehicles)
 
+  Создайте надежную, эффективную и специализированную для вашей индустрии платформу IoV на основе практического опыта EMQ, от теоретических знаний, таких как выбор протокола, до практических операций, таких как проектирование архитектуры платформы.
```
Comment (savonarola) [inline README-RU.md:87]:
```suggestion
  Создайте надежную, эффективную и специализированную для вашей индустрии платформу IoT на основе практического опыта EMQ, от теоретических знаний, таких как выбор протокола, до практических операций, таких как проектирование архитектуры платформы.
```

Comment (ieQu1) [reply README-RU.md:87]: The original document has IoV here.

File: README-RU.md (hunk 2 above)
Comment (ieQu1) [reply README-RU.md:66]: "Подпишитесь на"

**Review by ieQu1 — APPROVED** [review]

**Pull request merged**